src/style.css

```
@@ -0,0 +1,61 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 1200px;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars {
+  height: 16px;
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+  margin-top: 4px;
+  background: center / contain no-repeat url("./images/star.svg");
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--2 > .stars__star:nth-child(2),
+.stars--3 > .stars__star:nth-child(2),
+.stars--4 > .stars__star:nth-child(2),
+.stars--5 > .stars__star:nth-child(2) {
+  background-image: url("./images/star-active.svg");
+}
```

```suggestion
.stars--2 .stars__star:nth-child(2),
.stars--3 .stars__star:nth-child(2),
.stars--4 .stars__star:nth-child(2),
.stars--5 .stars__star:nth-child(2) {
  background-image: url("./images/star-active.svg");
}
```
Тебе тут не нужен лишний комбинатор